base/applications/mspaint/lang/vi-VN.rc

```diff
@@ -0,0 +1,228 @@
+/*
+ * PROJECT:     PAINT for ReactOS
+ * LICENSE:     LGPL
+ * FILE:        base/applications/mspaint/lang/vi-VN.rc
+ * PURPOSE:     VN-Vietnamese Language resource file
+ * TRANSLATORS: first times touched from en-US.rc file by Dummynosrev (Nguyễn Thanh Sơn)
+ */
+
+LANGUAGE LANG_VIETNAMESE, SUBLANG_VIETNAMESE_VN
```

Comment:
If you look at the CI results (where it says Äll checks have failed") you will see something like this:
`c:\reactos-cov\base\applications\mspaint\lang/vi-VN.rc(9) : error RC2147 : SUBLANGUAGE ID not a number`

To find the correct constant you can search for SUBLANG_VIETNAMESE: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=SUBLANG_VIETNAMESE

So this should say SUBLANG_VIETNAMESE_VIETNAM.

You can also find this kind of issue by building ROS locally with your changes.